workflows/templates/kebechet-job.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: kebechet-job
+  annotations:
+    thoth-station.ninja/template-version: 0.1.1
+  labels:
+    app: thoth
+    component: kebechet-job
+
+spec:
+  templates:
+    - name: kebechet-job
+      inputs:
+        parameters:
+        - name: WEBHOOK_PAYLOAD 
+      container:
+        # Chnage to thoth station kebechet master, repo.
+        image: "quay.io/saisankargochhayat/kebechet:master"
+        command: [sh, -c]
+        args: [
+            '
+            ./app.sh
+            ',
+          ]
+        env:
+          - name: PIPENV_NOSPIN
+            value: "1"
+          - name: PIPENV_COLORBLIND
+            value: "1"
+          - name: PIPENV_HIDE_EMOJIS
+            value: "1"
+          - name: KEBECHET_VERBOSE
+            value: "1"
+          - name: KEBECHET_SUBCOMMAND
+            value: "run-webhook"
+          - name: KEBECHET_PAYLOAD
+            value: "{{inputs.parameters.WEBHOOK_PAYLOAD}}"
+          - name: GITHUB_KEBECHET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: github-oauth-token
+                name: kebechet-secret
+          - name: GITLAB_KEBECHET_TOKEN
```

Comment:
@saisankargochhayat how do you wanna add the Gitlab cee ?
will this name is fine for gitlab cee variable ?
https://github.com/thoth-station/thoth-ops/blob/b3be9200456ce704e64d2cb7aa6eaf9b53764b9a/ansible/prepare_environment.yaml#L381

Reply:
@harshad16 I will test a the gitlab cee part later and add it along with the #336 pr. :smile: